actors/builtin/miner/miner_actor.go

```diff
@@ -351,10 +351,8 @@ func (a Actor) SubmitWindowedPoSt(rt Runtime, params *SubmitWindowedPoStParams)
 	}
 
 	partitionIndexes := bitfield.New()
-	if nv >= network.Version7 {
-		for _, partition := range params.Partitions {
-			partitionIndexes.Set(partition.Index)
-		}
+	for _, partition := range params.Partitions {
+		partitionIndexes.Set(partition.Index)
```

Comment:
This is now unused in this method, but this step is repeated within `deadline.RecordProvenSectors`

Reply:
Ah, thanks for the reminder. I wanted to avoid checking this via `IsSet` because that method is kind of slow.